### PR TITLE
Renaming sensilla

### LINKED ISSUE #1105: Rename all sensilla to use English names
Several types of sensilla defined in the ontology have a Latin name as their label (e.g. `sensillum campaniformium`, `sensillum basiconicum`, etc). Such Latin names don’t seem to be favored in the recent literature, English names being preferred (e.g. `campaniform sensillum`, `basiconic sensillum`, etc.).

It is proposed to make the terms for those sensilla more uniform by renaming:
* `sensillum campaniformium` (FBbt:00005183) to `campaniform sensillum`,
* `sensillum basiconicum` (FBbt:00005185) to `basiconic sensillum`,
* `sensillum coeloconicum` (FBbt:00005190) to `coeloconic sensillum`,
* `sensillum intermedia` (FBbt:00007358) to `intermediate sensillum`;
* `sensillum trichodeum` (FBbt:00005184) to `trichoid sensillum`,
* and all the more specific terms below accordingly (many of them actually already use an English name).

The Latin name would be kept as an exact synonym for each class.

### PR DESCRIPTION
**Pull request by gouttegd**
Rename all sensilla to use English adjectives instead of Latin ones.

closes #1105

### CODE REVIEW

File: src/ontology/fbbt-edit.obo [hunk 1]
```diff
@@ -83176,41 +83330,41 @@ creation_date: 2018-10-23T09:02:40Z
 
 [Term]
 id: FBbt:00047867
-name: sensillum campaniformium of dorsal radius D
+name: campaniform sensillum of dorsal radius D
 def: "Sensillum belonging to the anterior group of campaniform sensilla of the dorsal surface of the medial radius. There are four of these circular sensilla, which have a high profile and a socket (type 1). Each sensillum is approximately 4.5 micrometers in diameter." [FlyBase:FBrf0038304]
-synonym: "campaniform sensillum of dorsal radius D" EXACT [FlyBase:FBrf0038304]
 synonym: "d.Rad.D" EXACT [FlyBase:FBrf0038304]
-is_a: FBbt:00004346 ! sensillum campaniformium of dorsal radius Sc12
+synonym: "sensillum campaniformium of dorsal radius D" EXACT [FlyBase:FBrf0038304]
```
Comment (Clare72) [inline src/ontology/fbbt-edit.obo:83336]: not sure if this reference is still appropriate - if they were calling it "campaniform sensillum of dorsal radius D", they maybe weren't calling it "sensillum campaniformium of dorsal radius D", so probably best to get rid of references from newly added synonyms.

File: src/ontology/fbbt-edit.obo [hunk 2]
```diff
@@ -83176,41 +83330,41 @@ creation_date: 2018-10-23T09:02:40Z
...
 
 [Term]
 id: FBbt:00047868
-name: sensillum campaniformium of dorsal radius E
+name: campaniform sensillum of dorsal radius E
 def: "Sensillum belonging to the posterior group of campaniform sensilla of the dorsal surface of the medial radius. There are eight of these circular sensilla, which have a low profile and a socket (type 3). Each sensillum is approximately 5.0-5.5 micrometers in diameter." [FlyBase:FBrf0038304]
-synonym: "campaniform sensillum of dorsal radius E" EXACT [FlyBase:FBrf0038304]
 synonym: "d.Rad.E" EXACT [FlyBase:FBrf0038304]
-is_a: FBbt:00004346 ! sensillum campaniformium of dorsal radius Sc12
+synonym: "sensillum campaniformium of dorsal radius E" EXACT [FlyBase:FBrf0038304]
```
Comment (Clare72) [inline src/ontology/fbbt-edit.obo:83346]: remove reference

File: src/ontology/fbbt-edit.obo [hunk 3]
```diff
@@ -83176,41 +83330,41 @@ creation_date: 2018-10-23T09:02:40Z
...
 
 [Term]
 id: FBbt:00047869
-name: sensillum campaniformium of dorsal radius B
+name: campaniform sensillum of dorsal radius B
 def: "Sensillum belonging to the more proximal, posterior subgroup of Sc25 sensilla. There are seven of these elliptical sensilla, which have a low profile and a socket (type 6). Each sensillum is approximately 2.5-3.0 micrometers in length." [FlyBase:FBrf0038304]
-synonym: "campaniform sensillum of dorsal radius B" EXACT [FlyBase:FBrf0038304]
 synonym: "d.Rad.B" EXACT [FlyBase:FBrf0038304]
-is_a: FBbt:00004345 ! sensillum campaniformium of dorsal radius Sc25
+synonym: "sensillum campaniformium of dorsal radius B" EXACT [FlyBase:FBrf0038304]
```
Comment (Clare72) [inline src/ontology/fbbt-edit.obo:83356]: remove reference

File: src/ontology/fbbt-edit.obo [hunk 4]
```diff
@@ -83176,41 +83330,41 @@ creation_date: 2018-10-23T09:02:40Z
...
 
 [Term]
 id: FBbt:00047870
-name: sensillum campaniformium of dorsal radius C
+name: campaniform sensillum of dorsal radius C
 def: "Sensillum belonging to the more distal, anterior subgroup of Sc25 sensilla. There are around 17 of these circular sensilla, which have a low profile and no socket (type 4). Each sensillum is approximately 2.5-3.5 micrometers in diameter." [FlyBase:FBrf0038304]
-synonym: "campaniform sensillum of dorsal radius C" EXACT [FlyBase:FBrf0038304]
 synonym: "d.Rad.C" EXACT [FlyBase:FBrf0038304]
-is_a: FBbt:00004345 ! sensillum campaniformium of dorsal radius Sc25
+synonym: "sensillum campaniformium of dorsal radius C" EXACT [FlyBase:FBrf0038304]
```
Comment (Clare72) [inline src/ontology/fbbt-edit.obo:83366]: remove reference

File: src/ontology/fbbt-edit.obo [hunk 5]
```diff
@@ -85812,65 +85966,70 @@ creation_date: 2019-01-31T11:21:27Z
...
 creation_date: 2019-01-31T11:40:11Z
 
 [Term]
 id: FBbt:00048058
-name: mechanosensory neuron of leg sensillum campaniformium
-def: "Mechanosensory neuron with a dendrite in a sensillum campaniformium of the leg. Its axon terminals are found in the middle of the leg neuropil, relatively dorsal compared to other sensory neurons (Tsubouchi et al., 2017)." [FlyBase:FBrf0231054, FlyBase:FBrf0237124]
+name: mechanosensory neuron of leg campaniform sensillum
+def: "Mechanosensory neuron with a dendrite in a campaniform sensillum of the leg. Its axon terminals are found in the middle of the leg neuropil, relatively dorsal compared to other sensory neurons (Tsubouchi et al., 2017)." [FlyBase:FBrf0231054, FlyBase:FBrf0237124]
 synonym: "Lcs1N" NARROW [FlyBase:FBrf0237124]
-synonym: "leg campaniform sensillum primary neuron" EXACT [FlyBase:FBrf0237124]
```
Comment (Clare72) [inline src/ontology/fbbt-edit.obo:85849]: put this one back

File: src/ontology/fbbt-edit.obo [hunk 6]
```diff
@@ -85812,65 +85966,70 @@ creation_date: 2019-01-31T11:21:27Z
...
+synonym: "leg sensillum campaniformium primary neuron" EXACT [FlyBase:FBrf0237124]
```
Comment (Clare72) [inline src/ontology/fbbt-edit.obo:86005]: remove reference

File: src/ontology/fbbt-edit.obo [hunk 7]
```diff
@@ -88713,38 +88879,40 @@ creation_date: 2019-04-29T08:13:03Z
 
 [Term]
 id: FBbt:00048295
-name: haltere sensillum campaniformium
+name: haltere campaniform sensillum
 def: "Any campaniform sensillum that is part of the haltere. There are four groups of these, found on the dorsal and ventral sides of the proximal and middle parts." [FlyBase:FBrf0237124]
-synonym: "haltere campaniform sensillum" EXACT [FlyBase:FBrf0237124]
+synonym: "haltere sensillum campaniformium" EXACT [FlyBase:FBrf0237124]
```
Comment (Clare72) [inline src/ontology/fbbt-edit.obo:88884]: remove ref

**Review by Clare72 — CHANGES_REQUESTED** [review]
few minor things to fix

File: src/ontology/fbbt-edit.obo (hunk 1 above)
Comment (gouttegd) [reply src/ontology/fbbt-edit.obo:83336]: The reference is a paper from 1982 so there's little hope of checking the full text, but the available abstract uses "campaniform", so I agree it's not relevant to cite them as reference for a synonym they were likely not using at all.

**Review by Clare72 — APPROVED** [review]

**Pull request merged**